contrib/cirrus/setup.sh

```diff
@@ -28,3 +27,19 @@ if [[ "$CIRRUS_TASK_NAME" == "build_cross" ]]; then
 	# TODO: We can move this to quay.io if we reach rate-limits, hopefully that's not gonna happen for netavark
 	echo '  "rustembedded/cross" = "docker.io/rustembedded/cross"'  >> /etc/containers/registries.conf.d/000-shortnames.conf
 fi
+
+if [[ "$CIRRUS_TASK_NAME" == "podman_e2e" ]]; then
+	# build netavark
+	make
+	cp -v bin/netavark /usr/libexec/podman/
+	# clone podman upstream main
+	git clone https://github.com/containers/podman /var/tmp/go/src/github.com/containers/podman
+	rm /usr/bin/podman
```

Comment:
This is ripe for causing headaches.  An easier solution that also avoids compile time is to just grab Lokesh's podman-next COPR and `dnf update -y podman`.